cql3/expr/expression.cc

```diff
@@ -146,6 +146,44 @@ managed_bytes_opt get_value(const column_value& col, const column_value_eval_bag
 /// column that might be subscripted - e.g col1, col2, col3[sub1]
 using column_maybe_subscripted = std::variant<const column_value*, const subscript*>;
 
+/// Returns col's value from queried data.
+__attribute__((unused)) // For now mark as unused so that the code compiles, will be used soon
+static managed_bytes_opt get_value(const column_maybe_subscripted& col, const column_value_eval_bag& bag) {
```

Comment:
I hope this gets eventually folded into recursive evaluate(expression). I agree with this incremental approach.